RoadMap_SwiftUI/RoadMap_SwiftUI/View/LogInView.swift

```diff
@@ -0,0 +1,170 @@
+// LogInView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Экран входа
+struct LogInView: View {
+    // MARK: - Public properties
+
+    var body: some View {
+        topGradientView
+        Spacer()
+        VStack {
+            singInToggleView
+                .padding(.top, Constants.Numbers.paddingStandard)
+            loginStackView
+            signInButtonView
+            forgotPasswordButtonView
+            checkVerificationButtonView
+            Spacer()
+                .navigationBarTitleDisplayMode(.inline)
+                .navigationBarBackButtonHidden()
+        }
+    }
+
+    // MARK: - Private properties
+
+    @StateObject private var viewModel = LogInViewModel()
+    @FocusState private var loginIsFocused: Bool
+    @FocusState private var passwordIsFocused: Bool
```

Comment:
```suggestion
    @FocusState private var loginIsFocused
    @FocusState private var passwordIsFocused
```
можно не указывать тип